packages/mainsail/source/transaction-type.service.ts

```diff
@@ -17,7 +17,7 @@ export const trimHexPrefix = (type: string): string => {
 
 export class TransactionTypeService {
 	public static isTransfer(data: TransactionData): boolean {
-		return data.data === TransactionTypes.Transfer;
```

Comment:
remove it from the `TransactionTypes` list as well please, or turn that into an empty string instead

Reply:
should be good now